prepare_csv.py

```diff
@@ -0,0 +1,293 @@
+from io import BytesIO
+from zipfile import ZipFile
+from urllib.request import urlopen
+import csv
+
+# Preparation of a csv file with champions' games from the site "https://www.pgnmentor.com"
+
+tab = [

```

Comment:
"tab" is not a descriptive variable name. BTW, this is so long data, I'd recommend moving it outside Python, to a data file